.github/workflows/main.yml

```diff
@@ -21,7 +21,6 @@ jobs:
         run: |
           sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
           make deps
-          brew install conan
```

Comment:
I see that there is still a `conanfile.txt` in the repository. I guess it is not necessary anymore and can be removed?

Reply:
`conan` is being installed in that `make deps` call!